packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3509,6 +3527,8 @@ export default class Meeting extends StatelessWebexPlugin {
     }
     const previousSendShareStatus = this.mediaProperties.mediaDirection.sendShare;
 
+    this.setLocalShareTrack(stream);
```

Comment:
I see a small problem here: setLocalShareTrack() does more than just setting up onended listener. It also sends the 'media:ready' event to the app, so now we are sending this event before we've done any validation in `MeetingUtil.validateOption()` and before we've called  `MeetingUtil.updateTransceiver()` so if any of these 2 fail the last even that the app receives is 'media:ready', which doesn't seem right. I guess we should extract the setting up of onended out of setLocalShareTrack() and only do that bit here?